packages/app-project/stores/User/UserPersonalization/Notifications/helpers/getTalkUnreadConversationsIds.js

```diff
@@ -0,0 +1,31 @@
+import { talkAPI } from '@zooniverse/panoptes-js'
+
+export default async function getUnreadConversationsIds (authorization) {
+  if (!authorization) return undefined
+
+  let unreadConversationsIds = []
+
+  const getConversations = async (page = 1) => {
+    const query = {
+      unread: true,
+      page: page
+    }
+
+    const response = await talkAPI.get('/conversations', query, { authorization })
+    const { meta, conversations } = response?.body || {}
+
+    if (conversations && conversations.length > 0) {
+      unreadConversationsIds = unreadConversationsIds.concat(
+        conversations.map(conversation => conversation.id)
+      )
+    }
+
+    if (meta?.next_page) {
+      return getUnreadConversationsIds(meta.next_page)
```

Comment:
This won’t get the next page of results. It should fail the tests too, but everything is passing. 🤔

Reply:
🤦‍♂️ ! added a test for response with an additional page of resources